toolkits/global/packages/global-popup/js/popup.js

```diff
@@ -95,22 +96,48 @@ const Popup = class {
 
 	_calcPositioning() {
 		const distanceScrolled = document.documentElement.scrollTop;
-		const triggerMetrics = this._trigger.getBoundingClientRect();
+		const triggerMetricsCollection = this._trigger.getClientRects();
+		const triggerMetrics = (triggerMetricsCollection.length > 0) ? triggerMetricsCollection[0] : {top: 0, left: 0};
 		const offset = {
 			top: triggerMetrics.top + distanceScrolled,
 			left: triggerMetrics.left
 		};
+		const originalOffsetLeft = offset.left;
+
 		const arrow = this._content.querySelector(`.${this._arrowClass}`);
 		const windowWidth = document.documentElement.clientWidth;
 		const arrowHeight = 12;
 		const arrowWidth = 20;
+		const defaultOffsetLeft = 5;
+		const pagePadding = 32;
+		const mdBreakPoint = 768;
+		const lgBreakPoint = 1024;
```

Comment:
See above

Reply:
Yeah I'm not really sure they should be, breakpoints are breakpoints if they ever change they would have to be changed in the file passing them in or here and the breakpoints are the same across all imprints now I believe.